hooks/nostrProfile/useNostrProfile.ts

```diff
@@ -32,10 +45,13 @@ const useCachedNostrProfileEvent = (pubkey: string) => {
   return data;
 };
 
-export const useNostrProfile = () => {
-  const { pubkey } = useAuth();
-  const nostrProfileEvent = useNostrProfileEvent(pubkey ?? "");
-  const cachedNostrProfileEvent = useCachedNostrProfileEvent(pubkey ?? "");
+export const useNostrProfile = (pubkey?: string) => {
+  const { pubkey: loggedInPubkey } = useAuth();
+  // if no pubkey is provided, use the logged in user's pubkey
+  const nostrProfileEvent = useNostrProfileEvent(pubkey ?? loggedInPubkey);
+  const cachedNostrProfileEvent = useCachedNostrProfileEvent(
+    pubkey ?? loggedInPubkey,
+  );
```

Comment:
Could this be causing an issue with getting the avatar for the currently logged in user? Since we're renaming the `pubkey` const as the `loggedInPubkey`, won't they end up being the same?

Reply:
this change was to set this up to accept a pubkey argument on line 48, named `pubkey`. This caused a conflict with the pubkey variable returned by `useAuth`, so i explicitly renamed it to `loggedInPubkey`.

When `useNostrProfile` is used without passing in the optional `pubkey` argument, the `??` operator should fallback to using the value from `useAuth`, which is now renamed `loggedInPubkey`.

Reply:
i could have also just made a second hook named `useLoggedInNostrProfile` or something like that, and left it unchanged